indico/MaKaC/webinterface/tpls/ConfContributionListFilters.tpl

```diff
@@ -168,11 +168,11 @@
     var createMultiselect = function(place, kind){
         place.multiselect({
             selectedText: function(numChecked, numTotal, checkedItems){
-                return numChecked + " "+ kind + " selected";
+                return numChecked + " "+ kind + $T(" selected");
              },
-            noneSelectedText: "Select " + kind,
-            checkAllText: "All",
-            uncheckAllText: "None",
+            noneSelectedText: $T("Select ") + kind,
```

Comment:
This should be a formatted string:

    $T('Select {0}').format(kind)

Otherwise the translation might be weird depending on the language (imagine if there's a language where it's "<kind> select")